Recorded future solution 2.4

RecordedFutureOskbo opened the pull request:
Required items, please complete
   
   Change(s):
   - Handle 404 in enrichment playbook
   - Reduced size of image
   - Added new Sandbox enrichment playbook to solution

   Reason for Change(s):
   - Bug Fixes 
   - new playbook

  Testing Completed:
   - Yes

v-rbajaj commented:
Hi @manishkumar1991, can you please provide your feedback?

manishkumar1991 commented:
> Hi @manishkumar1991, can you please provide your feedback?

Review this PR

manishkumar1991 commented:
@RecordedFutureOskbo 
Kindly change the name of all arm template playbooks to azuredeploy.json ,    as per standard

RecordedFutureOskbo commented:
> @RecordedFutureOskbo Kindly change the name of all arm template playbook files to azuredeploy.json , as per standard

Renaming done
- Changed references in Solution_RecordedFuture.json
- Generated new maintemplate.json and zip
- Changed references in readme.md to new name 
- Corrected spelling mistakes in readme

v-rbajaj commented:
Looking into this PR. Will soon provide an update.

RecordedFutureOskbo commented:
@v-rbajaj , @manishkumar1991 , @v-atulyadav any feedback on this PR? We started to prepare for next version and don't want to  end up in merge conflict with our self. As feedback, the way the solution is packed together makes everything tangled up.

v-rbajaj commented:
Hi @RecordedFutureOskbo, the zip includes older versions please remove them, just keep the latest create ui and maintemplate.

RecordedFutureOskbo commented:
> Hi @RecordedFutureOskbo, the zip includes older versions please remove them, just keep the latest create ui and maintemplate.

Solution done! We pull all our data into the ThreatIntelligenceIndicator table. I'll try to export a representative dataset from our logic apps.

v-rbajaj commented on Solutions/Recorded Future/Playbooks/readme.md:
```diff
@@ -1,90 +1,274 @@
+
+There are two options for installing playbooks and starting automated threat response:
+
+- Installing the solution from from [Content Hub](https://portal.azure.com/#view/Microsoft_Azure_Marketplace/GalleryItemDetailsBladeNopdl/dontDiscardJourney~/true/id/recordedfuture1605638642586.recorded_future_sentinel_solution). (Recommended)
```
There are two "from" please remove one, thanks

RecordedFutureOskbo commented:
> Hi @RecordedFutureOskbo, the zip includes older versions please remove them, just keep the latest create ui and maintemplate. Also, can you please provide sample data?

I have added small amount of test data to the Solution. If you need data for specific testcases I can provide it to you.

v-rbajaj commented:
Hi @RecordedFutureOskbo, since this is a new solution, can you please package this solution with version 2.0.0?

RecordedFutureOskbo commented:
> Hi @RecordedFutureOskbo, since this is a new solution, can you please package this solution with version 2.0.0?

But this is version 2.4.0 of our solution, our current version in content hub is 2.3.0
![image](https://github.com/Azure/Azure-Sentinel/assets/104766953/ba1b6432-dd5d-4c80-8899-d2bc483a2787)
![image](https://github.com/Azure/Azure-Sentinel/assets/104766953/b9cbf8a7-7c0c-4dc3-8789-0dc59c9604fb)

v-rbajaj reviewed (approved):
Looks fine
